how-to/support-context-and-intents/client/src/interopbroker.ts

```diff
@@ -97,7 +97,7 @@ export function interopOverride(
 			// this logic runs in the provider so we are using it as a way of determining the root (so it works with root hosting and subdirectory based hosting if a url is not provided)
 			const url =
 				settings?.platformProvider?.intentPicker.url ||
-				window.location.href.replace("platform/provider.html", "common/windows/intents/picker.html");
+				"http://localhost:8080/common/windows/intents/picker.html";
```

Comment:
This will break live launch and it will mean they can't copy the sample to their own servers.

Reply:
package.js will update it.